### DISCUSSION TITLE
Bagaimana cara memunculkan legend pada ggplotly

### DISCUSSION THREAD

**Discussion by tu849035** [teamalgoritma/community#238] [Q&A] [answered]
Salam,

Saya ingin bertanya Bagaimana cara memunculkan legend pada ggplotly.
Untuk kode di bawah, saat dirun menggunakan ggplotly legend tidak muncul. Tetapi, saat run tanpa ggplotly legendnya muncul.
Mohon bantuan dan arahannya.

Videos_top_4 %>% 
  ggplot(aes(x = category_id,
             y = likesp,
             text = likesp)) +
  geom_boxplot(outlier.shape = NA, fill = "black" , col = "blue", alpha = 0.5 ) +
  geom_jitter((aes(size=commentp)) , col="green", alpha = 0.2) +
  labs(title = NULL,
       subtitle = "Entertainment, Music, Comedy, Howto and Style",
       x =NULL,
       y = "Likes per View",
       size = "Comment per View") +
  theme_minimal() +
  theme(legend.position = "top")

**Comment by triarm** [comment, marked as answer]
Hi @tu849035 ,

Sebelumnya untuk informasi yang ingin ditampilkan pada legend itu apa ya pa? Apakah jenis boxplotnya atau ukuran dari point jitternya?

**Reply by tu849035** [reply]
Ukuran dari point jitternya Mas. Kalau dirun tanpa ggplotly, legend mengenai ukuran point jitternya muncul.tampilannya seperti di bawah.

Tanpa ggplotly,
![image](https://user-images.githubusercontent.com/107348211/182589919-68e150ba-3c15-4c7b-b916-c8c76d256393.png)

Dengan ggplotly,
![image](https://user-images.githubusercontent.com/107348211/182590119-9f258b2c-25a0-4eef-98d4-f14345ad27b3.png)

**Reply by triarm** [reply]
Sejauh ini, by default untuk penampilan legend point size tidak didukung oleh ggplotly. Beberapa saran yang dapat kami sampaikan adalah:

1. Tidak perlu ditambahkan legend point size karena plot sudah interaktif dan dapat memunculkan nilai comment per view
2. Dapat digunakan skema gradasi warna untuk menunjukkan besaran dari nilai comment per view kemudian dimunculkan legend gradasi warnanya.

Demikian yang dapat kami sampaikan. Semoga membantu.
